File: RESOLUCOES/python/lista/01/ex013.py
# Faça um algoritmo que leia o salário de um funcionário, calcule e mostre o seu novo salário, com 15% de aumento.

salario = float(input('Digite o salário de funcionário para caclcular o aumento de 15%: '))

aumento = salario + salario * 0.15
print(f'O salário do funcionário com o aumento é de: R${aumento:.2f}')

File: resolucoes/java/lista01/src/main/java/org/example/Ex013.java
/*Faça um algoritmo que leia o salário de um funcionário, 
calcule e mostre o seu novo salário, com 15% de aumento. */

package org.example;

import java.util.Scanner;

public class Ex013 {
    public static void main(String[] args) {
        Ex013.main();
    }

    public static void main() {

        Scanner sc = new Scanner(System.in);
        System.out.print("Qual o salário do funcionário?: R$");
        Float salario = sc.nextFloat();
        System.out.printf("O salário do funcionário com o aumento de 15%% é %s", salario + salario / 100 * 15);
        sc.close();

    }

}
